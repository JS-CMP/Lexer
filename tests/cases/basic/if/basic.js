if (true) {
    console.log("This should always run")
}
if (false) {
    console.log("This should never run")
}
