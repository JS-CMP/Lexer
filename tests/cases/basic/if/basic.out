#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {if (JS::Any(true)){console["log"](JS::Any("This should always run"));};if (JS::Any(false)){console["log"](JS::Any("This should never run"));};}
